viite-UI/src/view/RoadAddressProjectForm.js

```diff
@@ -557,19 +557,21 @@
       });
 
       rootElement.on('click', '#cancelEdit', function(){
-        reOpenCurrent();
+        new GenericConfirmPopup('Haluatko tallentaa tekemäsi muutokset?', {
+          successCallback: function() {
```

Comment:
I didn't tested yet, but I don't know if the logic is correct here (at least looking at the names of functions doesn't seem correct). When we show the popup with the "Haluatko tallentaa tekemäsi muutokset?", we will close the project anyway, independently of the user choice. The only difference between the buttons is that one "save the changes and exit" and the other one just "exit the project without save". None of the options should go "next and enable interactions"